spec/haskell/src/SEL4/API/InvocationLabels/ARM.lhs

```diff
@@ -51,5 +51,5 @@ FIXME ARMHYP ARMPageMapIO is an inconsistant name (but coined by kernel team)
 >         | ARMVCPUAckVPPI
 #endif
 >         | ARMIRQIssueIRQHandler
+>         | ARMIRQIssueSGISignal
 >         deriving (Eq, Enum, Bounded, Show)
-
```

Comment:
last empty line removal?

Reply:
AFAIR, `git diff --check` complained. We used to have an extra empty line at the end for literal Haskell preprocessing, but that is unused now, so we don't need to do anything special there any more.